### PR TITLE
chore(logger): remove logger package, inline webpack logger

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
Remove logger package.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yes

**Summary**
- Remove logger package, inline webpack logger

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

**Comment by anshumanv** [comment]
@evilebottnawi rebased which removed your review, PTAL again. ✨

**Comment by rishabh3112** [comment]
Needs to fix conflicts, rest is fine.

**Comment by anshumanv** [comment]
Fixing in a bit. 👍

**Comment by anshumanv** [comment]
PTAL @evilebottnawi @rishabh3112, let's get this in. 🌟

**Comment by anshumanv** [comment]
Uhoh, taking look at tests.

File: packages/webpack-cli/lib/utils/logger.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const { logger } = require('@webpack-cli/logger');
+const { getLogger } = require('webpack/lib/logging/runtime');
```
Comment (sokra) [inline packages/webpack-cli/lib/utils/logger.js:1]: `webpack/lib/logging/runtime` should not be used here. Use `compiler.getInfrastructureLogger("webpack-cli")` instead.

Comment (anshumanv) [reply packages/webpack-cli/lib/utils/logger.js:1]: Will update

Comment (evilebottnawi) [reply packages/webpack-cli/lib/utils/logger.js:1]: @sokra we use logging before run webpack and after, and when we do other actions which don't need webpack (like `init`/`info`/`migrate`/`generate`), so we can't use infrastructure logger

**Pull request merged**